### PR TITLE
feat : Add Idyll clean option

### PR DESCRIPTION
**Pull request by amitbadala**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature update.

* **What is the current behavior?** (You can also link to an open issue here)
Addresses #533 

* **What is the new behavior (if this is a feature change)?** 
```Idyll clean``` option can be used to remove .idyll folder

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Other information**:

### CODE REVIEW

**Comment by amitbadala** [comment]
idyll-clean

**Comment by mathisonian** [comment]
Thanks @amitbadala! This is looking good. One thing I'd want to add before merging: 

The idyll folder can contain a secret token that is used to deploy projects to https://idyll.pub. I think this command should check for the existence of such a file (`.idyll/token`) and if it exists show a warning message along the lines of 

"This command will remove the idyll cache folder, including your idyll.pub token. Subsequent deploys to idyll.pub will receive a new URL. Are you sure you wish to continue?"

**Comment by amitbadala** [comment]
idyll-clean gif

File: packages/idyll-cli/bin/cmds/clean.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+};
+
+checkIfTokenExists = _ => {
+  return fs.pathExists(`${PATH}/token`);
+};
+
+confirmClean = async _ => {
+  const userResponse = await inquirer.prompt([
+    {
+      type: 'list',
```
Comment (mathisonian) [inline packages/idyll-cli/bin/cmds/clean.js]: I think type `confirm` would be slightly cleaner here

**Comment by mathisonian** [comment]
@amitbadala can you please avoid force pushing to PRs? You can push commits as you work and I'll squash them during the merge if necessary.

File: packages/idyll-cli/bin/cmds/clean.js (hunk 1 above)
Comment (amitbadala) [reply packages/idyll-cli/bin/cmds/clean.js]: done!!

**Comment by amitbadala** [comment]
> @amitbadala can you please avoid force pushing to PRs? You can push commits as you work and I'll squash them during the merge if necessary

my bad !.. would ensure that this is not repeated

File: packages/idyll-cli/bin/cmds/clean.js (hunk 1 above)
Comment (mathisonian) [reply packages/idyll-cli/bin/cmds/clean.js]: Hey sorry - to be clear I mean changing the `type: 'list'` line to `type: 'confirm'` and making any necessary updates (see inquirer docs (see docs https://github.com/SBoudrias/Inquirer.js#confirm---type-confirm)

File: packages/idyll-cli/bin/cmds/clean.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      }
+    } else {
+      removeIdyll();
+    }
+  } catch (e) {
+    console.log(e);
+  }
+};
+
+checkIfTokenExists = _ => {
```
Comment (mathisonian) [inline packages/idyll-cli/bin/cmds/clean.js]: I think this should be `const checkIfTokenExists = ...`

File: packages/idyll-cli/bin/cmds/clean.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  return fs.pathExists(`${PATH}/token`);
+};
+
+confirm = async _ => {
```
Comment (mathisonian) [inline packages/idyll-cli/bin/cmds/clean.js]: Needs declaration (e.g. `const`)

File: packages/idyll-cli/bin/cmds/clean.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      message: `This command will remove the idyll cache folder, including your idyll.pub token.
+      Subsequent deploys to idyll.pub will receive a new URL. 
+      Are you sure you wish to continue?`,
+      choices: ['Yes', 'No']
+    }
+  ]);
+  return userResponse.confirm;
+};
+
+removeIdyll = _ => {
```
Comment (mathisonian) [inline packages/idyll-cli/bin/cmds/clean.js]: Needs declaration (e.g. `const`)

File: packages/idyll-cli/bin/cmds/clean.js (hunk 1 above)
Comment (amitbadala) [reply packages/idyll-cli/bin/cmds/clean.js]: done !!

File: packages/idyll-cli/bin/cmds/clean.js (hunk 2 above)
Comment (amitbadala) [reply packages/idyll-cli/bin/cmds/clean.js]: done !!

File: packages/idyll-cli/bin/cmds/clean.js (hunk 3 above)
Comment (amitbadala) [reply packages/idyll-cli/bin/cmds/clean.js]: done !!

**Comment by mathisonian** [comment]
Looking great, thanks @amitbadala

**Pull request merged**